README.md

```diff
@@ -59,6 +59,20 @@ For ease of use in the the vagrant setup, we recommend you run the following:
 $ export KUBECONFIG="/etc/kubernetes/admin.conf"
 ```
 
+To verify the Kubernetes installation, run the following command:
+
+```bash
+$ kubectl get nodes
+NAME      STATUS    AGE
+master    Ready     22h
+node0     Ready     22h
+node1     Ready     22h
+node2     Ready     22h
+```
+
+#### NOTE: To see the version of Kubernetes (which will change based on latest official releases) simply do _kubectl version_
```

Comment:
Stylistic preferences: `_kubectl version_` should be ``kubectl version``, and `#### NOTE:` should be replaced with  `***NOTE:***` or some non-heading equivalent.